.github/workflows/health_checks.yml

```diff
@@ -319,7 +319,30 @@ jobs:
       - uses: ./.github/actions/setup_node
       - uses: ./.github/actions/restore_install_cache
       - run: npm run check:package-versions
-  update_or_publish_versions:
+
+  update_package_versions:
+    if: ${{ github.event_name == 'push' && (github.ref_name == 'main' || github.ref_name == 'hotfix') }}
+    needs:
+      - install
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # version 3.6.0
+      - uses: ./.github/actions/setup_node
+      - uses: ./.github/actions/restore_install_cache
+      - id: is_version_packages_commit
+        run: npx tsx scripts/is_version_packages_commit.ts "$GITHUB_EVENT_PATH" >> "$GITHUB_OUTPUT"
+      - name: Create or update Version Packages PR
+        # if this push is NOT merging a version packages PR, then we update/create the version packages PR
+        if: ${{ steps.is_version_packages_commit.outputs.is_version_packages_commit == 'false' }}
+        uses: changesets/action@f13b1baaa620fde937751f5d2c3572b9da32af23 # version 1.4.5
+        with:
+          # this should never be called, but if something happens and it does get called, this ensures that a premature publish won't happen
+          publish: echo Cannot publish during update version step
```

Comment:
should we set `createGithubReleases: false` here? just in case?

Reply:
good call, I'll update that.